dsl/src/commonMain/kotlin/com/freeletics/flowredux/dsl/util/AtomicInt.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 LifeTime GmbH. All rights reserved.
+ */
+
+package com.freeletics.flowredux.dsl.util
+
+public expect class AtomicInt(initialValue: Int) {
```

Comment:
Since we only use  `get()`, `increment()` and `decrement()` I would like to propose to limit the publi API only to this methods and maybe even rename the class to `AtomicCounter` to be more explicit about what this class will be about.

Reply:
And we could make this `internal`

Reply:
Redundant methods are removed. I'd love to make it `internal` but this isn't possible due to the typealias in jvmMain.

Reply:
We could in a follow up discussion talk if it makes sense to wrap jvm's `AtomicInt` around `AtomicCountet` (same way you did it for native targets so that we could mark everything as `internal`). But for now not a blocker!